### DISCUSSION TITLE
如何把调试界面左边窗口变大些？

### DISCUSSION THREAD

**Discussion by wxzvim** [ayamir/nvimdots#880] [Q&A] [answered]
![Uploading image.png…]()
如上图，，把左边窗口变大些，，按什么快捷键，，文档中没有找到

**Comment by Jint-lzxy** [comment]
Please wait for the images to be uploaded before starting a new discussion. Give GitHub some patience :)

**Reply by wxzvim** [reply]
OK

**Reply by Jint-lzxy** [reply]
You should re-upload that image. GitHub cancels image upload once u submitted ur discussion.

**Comment by wxzvim** [comment]
![image](https://github.com/ayamir/nvimdots/assets/110797126/5c90cce6-90cb-46d1-b794-645e26124894)
你看，，调试窗口，我就按了几次ctrl+n，，右边的代码窗口缩的好小，如何恢复原样？

**Comment by wxzvim** [comment]
每次按f6都这样，除非关闭这个文件才能恢复正常
![image](https://github.com/ayamir/nvimdots/assets/110797126/7a83a488-94de-4d1d-a507-dacdf9a122e5)

**Reply by Jint-lzxy** [reply, marked as answer]
[Wiki:Keybindings](https://github.com/ayamir/nvimdots/wiki/Keybindings) and
https://github.com/ayamir/nvimdots/blob/d425bfdc4f4d1f36d0ac1b452a58ef83f055ce20/lua/core/mapping.lua#L24-L27

**Reply by wxzvim** [reply]
OK

**Reply by wxzvim** [reply]
感谢
